apps/src/lib/script-editor/ScriptEditor.jsx

```diff
@@ -131,6 +132,15 @@ export default class ScriptEditor extends React.Component {
             style={styles.input}
           />
         </label>
+        <label>
+          Unit URL (Slug)
+          <input
+            type="text"
+            value={this.props.name}
```

Comment:
are you getting any js console errors? react warned me about setting value without adding an onChange method. if you are, please change this to defaultValue. (you might not be if react is smart enough to realize that onchange isn't needed for a `disabled` input)

Reply:
I checked and there is no error or warning